src/Sylius/Bundle/MailerBundle/Resources/config/services.xml

```diff
@@ -46,11 +49,22 @@
             <argument>%sylius.mailer.sender_address%</argument>
         </service>
 
-        <service id="sylius.email_sender.adapter.twig_swiftmailer" class="%sylius.email_sender.adapter.twig_swiftmailer.class%">
-            <argument type="service" id="sylius.mailer.default_settings_provider" />
-            <argument type="service" id="mailer" />
+        <service id="sylius.email_renderer.adapter.abstract" class="%sylius.email_renderer.adapter.abstract.class%" abstract="true">
```

Comment:
I'm not sure that all those new services should be public as they will be only used internally by `Sender` class...